api/v1beta2/foundationdbcluster_types.go

```diff
@@ -866,6 +866,10 @@ type FoundationDBClusterAutomationOptions struct {
 	// +kubebuilder:validation:Enum=Replace;ReplaceTransactionSystem;Delete
 	// +kubebuilder:default:=ReplaceTransactionSystem
 	PodUpdateStrategy PodUpdateStrategy `json:"podUpdateStrategy,omitempty"`
+
+	// UseManagementAPI for using the management APIs that FDB provides.
+	// ex: Using FDB key to get connection string.
```

Comment:
```suggestion
	// UseManagementAPI defines if the operator should make use of the management API instead of
	// using fdbcli to interact with the FoundationDB cluster.
```